doc/releases/changelog-0.9.0.md

```diff
@@ -2,161 +2,175 @@
+      return jnp.sin(circuit(x ** 2))
   ```
 
-  `pipeline` can also be used to specify different pass pipelines for different parts of the
-  same qjit-compiled workflow:
+  `pipeline` can also be used to specify different pass pipelines for different parts of the same
+  qjit-compiled workflow:
 
   ```python
-    my_pipeline = {
-        "cancel_inverses": {},
-        "my_circuit_transformation_pass": {"my-option" : "my-option-value"},
-    }
-
-    my_other_pipeline = {"cancel_inverses": {}}
-
-    @qjit
-    def fn(x):
-        circuit_pipeline = pipeline(my_pipeline)(circuit)
-        circuit_other = pipeline(my_other_pipeline)(circuit)
-        return jnp.abs(circuit_pipeline(x) - circuit_other(x))
-  ```
+  my_pipeline = {
+      "cancel_inverses": {},
+      "my_circuit_transformation_pass": {"my-option" : "my-option-value"},
+  }
+
+  my_other_pipeline = {"cancel_inverses": {}}
 
-  For a list of available passes, please see the
-  [catalyst.passes module documentation](https://docs.pennylane.ai/projects/catalyst/en/stable/code/__init__.html#module-catalyst.passes).
+  @qjit
+  def fn(x):
+      circuit_pipeline = pipeline(my_pipeline)(circuit)
+      circuit_other = pipeline(my_other_pipeline)(circuit)
+      return jnp.abs(circuit_pipeline(x) - circuit_other(x))
+  ```
 
-  The pass pipeline order and options can be configured *globally* for a qjit-compiled
-  function, by using the `circuit_transform_pipeline` argument of the :func:`~.qjit`
-  decorator.
+  The pass pipeline order and options can be configured *globally* for a qjit-compiled function, by
+  using the `circuit_transform_pipeline` argument of the :func:`~.qjit` decorator.
 
   ```python
-    my_passes = {
-        "cancel_inverses": {},
-        "my_circuit_transformation_pass": {"my-option" : "my-option-value"},
-    }
-
-    @qjit(circuit_transform_pipeline=my_passes)
-    def fn(x):
-        return jnp.sin(circuit(x ** 2))
-  ```
+  my_passes = {
+      "cancel_inverses": {},
+      "my_circuit_transformation_pass": {"my-option" : "my-option-value"},
+  }
 
-  Global and local (via `@pipeline`) configurations can coexist, however local pass pipelines
-  will always take precedence over global pass pipelines.
+  @qjit(circuit_transform_pipeline=my_passes)
+  def fn(x):
+      return jnp.sin(circuit(x ** 2))
+  ```
 
-  Available MLIR passes are now documented and available within the
-  [catalyst.passes module documentation](https://docs.pennylane.ai/projects/catalyst/en/stable/code/__init__.html#module-catalyst.passes).
+  Global and local (via `@pipeline`) configurations can coexist, however local pass pipelines will
+  always take precedence over global pass pipelines.
 
-  The `pipeline` function is available as `catalyst.pipeline` and `catalyst.passes.pipeline`.
+  The available MLIR passes are listed and documented in the
+  [passes module documentation](https://docs.pennylane.ai/projects/catalyst/en/stable/code/__init__.html#module-catalyst.passes).
```

Comment:
Do we want to keep the "the pipeline function is available as catalyst.pipeline and catalyst.passes.pipeline" line? I'm fine with removing it too, but personally I think changelog should prioritize being informative and as encompassing as possible, over being concise.

Reply:
I think we can put it back in, I may have accidentally removed it in a merge resolution!

Reply:
@paul0403 in user facing documentation, I would keep it simple and _only specify_ one location for a function to avoid complication. So let's just keep it as `catalyst.pipeline` in our user-facing docs/tutorials

Reply:
> @paul0403 in user facing documentation, I would keep it simple and _only specify_ one location for a function to avoid complication. So let's just keep it as `catalyst.pipeline` in our user-facing docs/tutorials

Do we consider changelog as user facing?

Reply:
> Do we consider changelog as user facing?

Yes 😎

Reply:
In that case I'm fine with removing this line!

Reply:
Okay I've left it out.